core/src/main/resources/jenkins/monitor/EndOfLifeAlpine315AdminMonitor/description.properties

```diff
@@ -0,0 +1 @@
+blurb = Warns administrators when Alpine 3.15 is approachinig end of life and will be unsupported
```

Comment:
typo throughout
```suggestion
blurb=Warns administrators when Alpine 3.15 is approaching end of life and will be unsupported
```

Reply:
Thanks.  Fixed in 1753bee8b73a5735b2a2b1f74a1f1a627a5aa9f0 .  The spaces around the `=` sign exist in 93 other locations that use `blurb`.  1753bee8b73a5735b2a2b1f74a1f1a627a5aa9f0 did not change any of those locations, assuming that they are harmless.